main.tf

```diff
@@ -33,6 +33,7 @@ data "template_file" "nomad_job_minio" {
     host_volume               = var.nomad_host_volume
     image                     = var.container_image
     service_name              = var.service_name
+    consul_tags               = join(",", var.consul_tags)
```

Comment:
Can you update README with this new variable?

Reply:
Yes